shard-operator/src/main/kubernetes/minikube.yml

```diff
@@ -1,115 +0,0 @@
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: shard-operator
-rules:
-  - apiGroups:
-      - com.redhat.service.bridge
-    resources:
-      - bridgeexecutors
-      - bridgeexecutors/status
-      - bridgeingresses
-      - bridgeingresses/status
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - delete
-      - patch
-      - update
-  - apiGroups:
-      - apiextensions.k8s.io
-    resources:
-      - customresourcedefinitions
-    verbs:
-      - get
-      - list
-      - watch
-  - apiGroups:
-      - ""
-      - extensions
-      - apps
-    resources:
-      - deployments
-      - services
-      - namespaces
-      - secrets
-      - configmaps
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - delete
-      - patch
-      - update
-  - apiGroups:
-      - networking.k8s.io
-    resources:
-      - ingresses
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - delete
-      - patch
-      - update
-  - apiGroups:
-      - monitoring.coreos.com
-    resources:
-      - servicemonitors
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - delete
-      - patch
-      - update
-  - apiGroups:
-      - eventing.knative.dev
-    resources:
-      - brokers
-      - triggers
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - delete
-      - patch
-      - update
-  - apiGroups:
-      - security.istio.io
-    resources:
-      - requestauthentications
-      - authorizationpolicies
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - delete
-      - patch
-      - update
-  # add here the core resources we need to manage
-  #- apiGroups:
-  #    - ""
-  #  resources:
-  #    - secrets
-  #  verbs:
-  #    - "*"
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: shard-operator
-roleRef:
-  kind: ClusterRole
-  apiGroup: rbac.authorization.k8s.io
-  name: shard-operator
-subjects:
-  - kind: ServiceAccount
-    name: shard-operator
```

Comment:
was this not used?

Reply:
no.. its used. The `shard-operator` service-account would bind to the operator deployment object.

Reply:
Found it, thanks

Reply:
The "Build and Test Operator" step states `Error from server (NotFound): deployments.apps "shard-operator" not found`.

Is this related?

Reply:
yes, It will be fixed with changes in this PR